src/Bundle/Resources/config/services/console.xml

```diff
@@ -15,9 +15,10 @@
     <services>
         <defaults public="true" />
 
-        <service id="sylius.console.command.resource_debug" class="Sylius\Bundle\ResourceBundle\Command\DebugResourceCommand">
+        <service id="Sylius\Bundle\ResourceBundle\Command\DebugResourceCommand">
             <argument type="service" id="sylius.resource_registry" />
             <tag name="console.command" />
         </service>
+        <service id="sylius.console.command.resource_debug" alias="Sylius\Bundle\ResourceBundle\Command\DebugResourceCommand" />
```

Comment:
@loic425 @lchrusciel Changes like this one could technically be a B/C break.  You're changing a service ID from a definition to an alias.  Granted, with this specific case, it's probably not a big deal, but if someone were doing something in a compiler pass, a `$container->getDefinition('sylius.console.command.resource_debug')` call would no longer work.